server/app/services/applicant/ApplicantData.java

```diff
@@ -78,6 +78,38 @@ public Optional<String> getApplicantName() {
     return Optional.of(firstName);
   }
 
+  public Optional<String> getApplicantFirstName() {
```

Comment:
question (blocking): Do we need to add tests to `ApplicantDataTest` for these new methods?

Reply:
Tested these in both the controller and accountRepository.

Reply:
Got it thanks!